branch.map

```diff
@@ -1,3 +1,5 @@
-branch      sqs_queue_url                                                                                    s3_destination
-main        https://sqs.eu-west-2.amazonaws.com/654654490827/ukceh_fdri_staging_metadata_update_queue.fifo   s3://ukceh-fdri-staging-metadata-state/data
-expt        https://sqs.eu-west-1.amazonaws.com/293385631482/ceh-dev-fuseki.fifo                             s3://expt-eks-cluster/data/ceh/dev
+branch      sqs_queue_url                                                                                       s3_destination
+dev         https://sqs.eu-west-2.amazonaws.com/654654490827/ukceh_fdri_staging_dev_metadata_update_queue.fifo  s3://ukceh-fdri-staging-dev-metadata-state/data                                                                                                        s3://ukceh-fdri-staging-dev-metadata-state/data
+main        https://sqs.eu-west-2.amazonaws.com/654654490827/ukceh_fdri_staging_dev_metadata_update_queue.fifo  s3://ukceh-fdri-staging-dev-metadata-state/data                                                                                                        s3://ukceh-fdri-staging-dev-metadata-state/data
```

Comment:
The S3 destination seems to be duplicated, is that deliberate? Seems like the publish code would ignore so not harmful but ...

Reply:
It's the second copy of ` s3://ukceh-fdri-staging-dev-metadata-state/data` far to the right of both lines that confused me.